AddShippingZone.tsx

```diff
@@ -1,18 +1,138 @@
-import React from "react";
-import { Button, SafeAreaView, Text, View } from "react-native";
+import React, { useState } from "react";
+import {
+  Linking,
+  Platform,
+  Pressable,
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  Text,
+  TextInput,
+  View,
+} from "react-native";
 import { useNavigation } from "@react-navigation/native";
+import CheckBox from "@react-native-community/checkbox";
+import { ToolbarActionButton } from "./ToolbarActionButton";
 
 const AddShippingZone = () => {
   const navigation = useNavigation();
 
+  const [isLimitEnabled, setLimitEnabled] = useState(false);
+
+  React.useEffect(() => {
+    navigation.setOptions({
+      headerRight: () =>
+        ToolbarActionButton({
+          label: "Save",
+          onPress: () => console.log("Toolbar action button pressed"),
+        }),
+    });
+  }, [navigation]);
+
+  const _renderPostCodes = function () {
```

Comment:
[Q] Does the `_` prefix has any react meaning?

Reply:
I thought inner functions were normal functions and outter functions where export functions 😅

Reply:
True, I'll remove the prefix in a next PR 🙂